### ISSUE TITLE
Can't bypass new challenge

### ISSUE THREAD

**Issue by andress134** [nspin/cloudpiercer#4]
Hello sir, your module is ouptdate can't bypass new cloudflare challenge

**Comment by nspin** [comment]
Yes, they've updated the challenge. I haven't taken a look yet to determine whether this project's current approach fits the new challenge.
